enemies/main.json

```diff
@@ -802,7 +802,8 @@
       "mult": {},
       "areas": [
         "brinstar",
-        "crateria"
+        "crateria",
+        "maridia"
```

Comment:
Some Yapping Maws were placed in Upper Norfair too, I think?

Reply:
Indeed, in "Spiky Acid Snakes Tunnel"